package.json

```diff
@@ -38,7 +38,7 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "start-ganache": "ganache-cli -i 1984 --mnemonic 'hello unlock save the web'",
     "postinstall": "./scripts/postinstall.sh",
-    "deploy-lock": "cd smart-contracts && npm run deploy -- --network local",
+    "deploy-lock": "cd smart-contracts && npm run deploy",
```

Comment:
This should be unnecessary / have 0 impact

Reply:
Why do we not need to specify the network here?

Reply:
I'm trying to simplify the truffle-config.js down the line.  What value does network `local` offer now - it's effectively being merged with development which means we should be able  to just use default values again.

Reply:
I see. "Local" was only added when "development" was needed for solidity coverage. So if we remove local, and don't specify a network, "development" is used. Does this sound right?

Reply:
Yea.  I'm going to revert this for now to simplify the PR

Reply:
Sounds good, but I like the direction this is going, towards simplification!